src/Middleware/Rewrite/src/RedirectToHttpsRule.cs

```diff
@@ -18,10 +18,11 @@ public virtual void ApplyRule(RewriteContext context)
             if (!context.HttpContext.Request.IsHttps)
             {
                 var host = context.HttpContext.Request.Host;
-                if (SSLPort.HasValue && SSLPort.Value > 0)
+                int port;
+                if (this.SSLPort.HasValue && (port = this.SSLPort.GetValueOrDefault()) > 0)
```

Comment:
There are a lot of format changes we don't use in the aspnetcore repo (`this` for example). Could you remove that?

Reply:
Done!

This repo should have .editorconfig files to validate and enforce these rules.